Add (generated) motion history (#17)
thunderbird/council-docs: MOTIONS.md

# Council Motion History

> Council motions are used for formal decision making in the Thunderbird Council. See the [bylaws](https://github.com/thunderbird/council-docs/blob/main/BY_LAWS.md#decision-making-process) for more information. This document captures the formal decisions that have been made, which go beyond informal disucssion that can happen in addition.

Reviewer: ```suggestion
Council motions are used for formal decision making in the Thunderbird Council. See the [bylaws](https://github.com/thunderbird/council-docs/blob/main/BY_LAWS.md#decision-making-process) for more information. This document captures the formal decisions that have been made, which go beyond informal discussion that can happen in addition.

Note that old motions may have been made under processes that don't follow the current bylaws.
```

individuals may also get involved in writing or commenting on the proposed specs.”</p></td><td valign="top">2023-06-12<br>Proposed: Ben<br><br>Seconded: Berna</td><td valign="top">Yes: Ben, Berna, Danny, John, Micah</td><td valign="top"></td></tr>
<tr><td valign="top"><p>Propose to record council meetings with a retention period of [TBD].</p></td><td valign="top">2023-03-27<br>Proposed: Danny<br><br>Seconded: Ben</td><td valign="top">Yes: Berna, Danny, John, Micah, Patrick</td><td valign="top"></td></tr>
<tr><td valign="top"><p>Approve <a href="https://nextcloud.thunderbird.net/f/10800">MZLA/Thunderbird FY23 Budget</a>.</p></td><td valign="top">2023-02-06<br>Proposed: Patrick<br><br>Seconded: Danny, John</td><td valign="top">Yes: Berna, Danny, John, Micah, Patrick<br><br>No: Ben</td><td valign="top"></td></tr>
</table>

## 2021–2022

**Term**: October 28, 2021 to December 20, 2022

**Council Members:**
- Berna Alp (Chair)
- Andrei Hajdukewycz (Secretary)
- Ben Bucksch
- Dirk Steinmetz
- Magnus Melin
- Patrick Cloke
- Philipp Kewisch

<table><tr><th>Motion</th><th>Details</th><th>Votes</th><th>Statements</th>

> <tr><td valign="top"><p>Allow new council members to see previous term’s Council meeting notes, unless there’s a specific, localized issue regarding a specific new Councillor. Such an issue must be proposed by 4 Councilors.</p></td><td valign="top">2022-12-05<br>Proposed: Ben<br><br>Seconded: Berna</td><td valign="top">Yes: Andrei, Ben, Berna, Dirk, Patrick</td><td valign="top"></td></tr>

Reviewer 2: ```suggestion
<tr><td valign="top"><p>Allow new council members to see previous term’s Council meeting notes, unless there’s a specific, localized issue regarding a specific new Councilor. Such an issue must be proposed by 4 Councilors.</p></td><td valign="top">2022-12-05<br>Proposed: Ben<br><br>Seconded: Berna</td><td valign="top">Yes: Andrei, Ben, Berna, Dirk, Patrick</td><td valign="top"></td></tr>
```

<tr><td valign="top"><p>Create a new public repository at https://github.com/thundernest/council-docs with the description 'Official repository for organizational matters regarding the Thunderbird council, the elected body governing the Thunderbird project' and give all councilors that both have GitHub accounts and are interested administrative access to that repository. The repository features 'Wikis', 'Issues', 'Sponsorships', 'Projects', 'Discussions' must be disabled. The repository's main branch and settings must not be changed by anyone, unless there is an explicit council motion requesting the specific change. Any councilor may revert and/or publicly report changes that were not performed in accordance with a council motion.</p></td><td valign="top">2022-04-11<br>Proposed: Dirk<br><br>Seconded: Magnus</td><td valign="top">Yes: Andrei, Ben, Berna, Dirk, Magnus, Patrick</td><td valign="top"></td></tr>

> <tr><td valign="top"><p>[CONFIDENTIAL - This motion was redacted/reworded for publishing]<br>Start negotiations and prepare contracts for MZLA aquire K-9 including brand assets, including donation cash flow. The full contracts are to be presented to the council for final approval. Negotiators shall immediately communicate any unforeseen details.</p></td><td valign="top">2022-02-28<br>Proposed: Berna<br><br>Seconded: Ben</td><td valign="top">Yes: Andrei, Ben, Berna, Dirk<br><br>Abstain: Magnus, Patrick, Philipp</td><td valign="top"></td></tr>

Reviewer 2: ```suggestion
<tr><td valign="top"><p>[CONFIDENTIAL - This motion was redacted/reworded for publishing]<br>Start negotiations and prepare contracts for MZLA acquire K-9 including brand assets, including donation cash flow. The full contracts are to be presented to the council for final approval. Negotiators shall immediately communicate any unforeseen details.</p></td><td valign="top">2022-02-28<br>Proposed: Berna<br><br>Seconded: Ben</td><td valign="top">Yes: Andrei, Ben, Berna, Dirk<br><br>Abstain: Magnus, Patrick, Philipp</td><td valign="top"></td></tr>
```

<tr><td valign="top"><p>Have a community discussion asking about mobile clients in general, but not mentioning K-9, asking: How would they like the mobile client to look like? Native client? What is important for you in it?</p></td><td valign="top">2022-01-31<br>Proposed: Dirk<br><br>Seconded: Unknown</td><td valign="top">Yes: Andrei, Ben, Berna, Dirk<br><br>Abstain: Magnus, Patrick, Philipp</td><td valign="top"></td></tr>
<tr><td valign="top"><p>As general direction, but not as business strategy:
"Thunderbird: Secure and build on position by
(a) ensuring continued product/market fit (e.g., mobile, enterprise), and

> (b) ensuring financial health by building/integrating value added services and monetising

Reviewer 2: ```suggestion
(b) ensuring financial health by building/integrating value added services and monetizing
```

    through subscription or enhanced donation mechanics"</p></td><td valign="top">2022-01-31<br>Proposed: Berna<br><br>Seconded: Andrei</td><td valign="top">Yes: Andrei, Berna, Magnus, Patrick, Philipp<br><br>No: Ben, Dirk</td><td valign="top"></td></tr>
<tr><td valign="top"><p>The Council requests that the MZLA board perform due diligence with the intent to acquire K9 mail. Once the board has done this, they will report their findings to the Council, and a separate motion will decide how and whether the acquisition will be finalized.</p></td><td valign="top">2022-01-10<br>Proposed: Andrei<br><br>Seconded: Ben</td><td valign="top">Yes: Andrei, Ben, Berna, Dirk, Patrick, Philipp<br><br>No: Magnus</td><td valign="top"></td></tr>
<tr><td valign="top"><p>Approve and adopt the 2022 Budget with the intent to review it.</p></td><td valign="top">2022-01-10<br>Proposed: Berna<br><br>Seconded: Magnus</td><td valign="top">Yes: Andrei, Ben, Berna, Dirk, Magnus, Patrick, Philipp</td><td valign="top"></td></tr>

> <tr><td valign="top"><p>[CONFIDENTIAL - This motion was redacted/reworded for publishing]<br>A motion related to one time bonus payout with specfic amounts/terms.</p></td><td valign="top">2022-01-10<br>Proposed: Berna<br><br>Seconded: Ben</td><td valign="top">Yes: Ben, Berna<br><br>No: Dirk, Patrick, Philipp<br><br>Abstain: Andrei, Magnus</td><td valign="top"></td></tr>

Reviewer 2: ```suggestion
<tr><td valign="top"><p>[CONFIDENTIAL - This motion was redacted/reworded for publishing]<br>A motion related to onetime bonus payout with specific amounts/terms.</p></td><td valign="top">2022-01-10<br>Proposed: Berna<br><br>Seconded: Ben</td><td valign="top">Yes: Ben, Berna<br><br>No: Dirk, Patrick, Philipp<br><br>Abstain: Andrei, Magnus</td><td valign="top"></td></tr>
```

<tr><td valign="top"><p>[CONFIDENTIAL - This motion was redacted/reworded for publishing]<br>A motion to contract design system review from Ura Design</p></td><td valign="top">2021-09-27<br>Proposed: Ryan<br><br>Seconded: Patrick</td><td valign="top">Yes: Berna, Christopher, Dirk, Magnus, Patrick, Philipp, Ryan</td><td valign="top"></td></tr>
<tr><td valign="top"><p>For the current election cycle, make the mailing list elections run on publicly readable on discuss.thunderbird.net (Topicbox). Posting shall remain limited to people in the electorate, with the existing restrictions outside of the discussion phase.</p></td><td valign="top">2021-09-27<br>Proposed: Dirk<br><br>Seconded: Philipp</td><td valign="top">Yes: Berna, Dirk<br><br>No: Christopher, Magnus, Patrick, Ryan</td><td valign="top"></td></tr>
<tr><td valign="top"><p>[CONFIDENTIAL - This motion was redacted/reworded for publishing]<br>Team re-org (August 2021): Promotion to team lead</p></td><td valign="top">2021-09-13<br>Proposed: Magnus<br><br>Seconded: Unknown</td><td valign="top">Yes: Berna, Christopher, Magnus, Patrick, Ryan<br><br>Abstain: Dirk, Philipp</td><td valign="top"></td></tr>

> <tr><td valign="top"><p>Team re-org (August 2021): Change the reporting structure under magnus to be as in the chart, as of September 1.</p></td><td valign="top">2021-09-13<br>Proposed: Ryan<br><br>Seconded: Berna</td><td valign="top">Yes: Berna, Christopher, Magnus, Patrick, Ryan<br><br>Abstain: Dirk, Philipp</td><td valign="top"></td></tr>

Reviewer 2: ```suggestion
<tr><td valign="top"><p>Team re-org (August 2021): Change the reporting structure under Magnus to be as in the chart, as of September 1.</p></td><td valign="top">2021-09-13<br>Proposed: Ryan<br><br>Seconded: Berna</td><td valign="top">Yes: Berna, Christopher, Magnus, Patrick, Ryan<br><br>Abstain: Dirk, Philipp</td><td valign="top"></td></tr>
```

<tr><td valign="top"><p>[CONFIDENTIAL - This motion was redacted/reworded for publishing]<br>Change the title of Ryan Sipes to "Community and Business Development Manager". Define his new responsibilities and salary.</p></td><td valign="top">2019-03-24<br>Proposed: Magnus<br><br>Seconded: Philipp</td><td valign="top">Yes: 5</td><td valign="top"></td></tr>
</table>

## 2018–2019

**Term**: March 31, 2018 to March 21, 2019

**Council Members:**
- Philipp Kewisch (Chair)
- Ryan Sipes (Treasurer)
- Wayne Mery (Secretary)
- Jörg Knobloch
- Magnus Melin
- Patrick Cloke
- Philippe Lieser

<table><tr><th>Motion</th><th>Details</th><th>Votes</th><th>Statements</th>
<tr><td valign="top"><p>[CONFIDENTIAL - This motion was redacted/reworded for publishing]<br>A motion related to hiring an employee.</p></td><td valign="top">2019-03-21<br>Proposed: Magnus<br><br>Seconded: Wayne</td><td valign="top">Yes: 5<br><br>No: 1</td><td valign="top"></td></tr>

> <tr><td valign="top"><p>As clarification of the Council decision of 2018-01-31 stating that only two people of the same organisation/association/unit or company can be nominated for election to the Thunderbird Council; The 'Mozilla Corporation' and the 'Mozilla Foundation' which serves as legal home of the independent 'Thunderbird Project' (and hence fiducially contracts staff for the project) are (considered) separate companies for purposes of "professional affiliation".</p></td><td valign="top">2019-03-13<br>Proposed: Wayne<br><br>Seconded: Patrick</td><td valign="top">Yes: Berna, Magnus, Philipp, Ryan<br><br>Abstain: Jörg, Patrick, Wayne</td><td valign="top"></td></tr>

Reviewer 2: ```suggestion
<tr><td valign="top"><p>As clarification of the Council decision of 2018-01-31 stating that only two people of the same organization/association/unit or company can be nominated for election to the Thunderbird Council; The 'Mozilla Corporation' and the 'Mozilla Foundation' which serves as legal home of the independent 'Thunderbird Project' (and hence fiducially contracts staff for the project) are (considered) separate companies for purposes of "professional affiliation".</p></td><td valign="top">2019-03-13<br>Proposed: Wayne<br><br>Seconded: Patrick</td><td valign="top">Yes: Berna, Magnus, Philipp, Ryan<br><br>Abstain: Jörg, Patrick, Wayne</td><td valign="top"></td></tr>
```

<tr><td valign="top"><p>Sending any European council member to FOSDEM, plus Ryan as community manager. Travel costs max 1250 EUR for European travelers, and 2250 EUR for international travelers</p></td><td valign="top">2019-01-08<br>Proposed: Philipp<br><br>Seconded: Wayne</td><td valign="top">Yes: 4</td><td valign="top"></td></tr>
<tr><td valign="top"><p>Collaborate with BenB on getting this into account creation (non-exclusively), but no contract.</p></td><td valign="top">2018-11-20<br>Proposed: Ryan<br><br>Seconded: Wayne</td><td valign="top">Yes: 4</td><td valign="top"></td></tr>
<tr><td valign="top"><p>[CONFIDENTIAL - This motion was redacted/reworded for publishing]<br>A motion related to accepting a % rate to negotiate a partnership.</p></td><td valign="top">2018-07-18<br>Proposed: Ryan<br><br>Seconded: Unknown</td><td valign="top">Yes: 7</td><td valign="top"></td></tr>

> <tr><td valign="top"><p>[CONFIDENTIAL - This motion was redacted/reworded for publishing]<br>Spend an amiunt to pay a contractor for fixing the specific security bug(s). We will attempt to use the SOS funds, but if that is not possible we will use Thunderbird funds.</p></td><td valign="top">2018-06-06<br>Proposed: Unknown<br><br>Seconded: Unknown</td><td valign="top">Yes: Majority (unknown)</td><td valign="top"></td></tr>

Reviewer 2: ```suggestion
<tr><td valign="top"><p>[CONFIDENTIAL - This motion was redacted/reworded for publishing]<br>Spend an amount to pay a contractor for fixing the specific security bug(s). We will attempt to use the SOS funds, but if that is not possible we will use Thunderbird funds.</p></td><td valign="top">2018-06-06<br>Proposed: Unknown<br><br>Seconded: Unknown</td><td valign="top">Yes: Majority (unknown)</td><td valign="top"></td></tr>
```
